doc/requirements.rst

```diff
@@ -0,0 +1,119 @@
+=====================
+Software Requirements
+=====================
+
+.. contents:: `Contents`
+    :depth: 3
+    :local:
+
+---------------------------------
+Requirements for mlx.traceability
+---------------------------------
+
+.. item:: RQT-TRACEABILITY A plugin for sphinx documentation system, adding traceability
+
+    System shall implement a plugin for the sphinx documentation system. It shall add tracebility within
+    the documentation.
+
+.. item:: RQT-ITEMIZE Allow splitting the documentation in parts
+    :depends_on: RQT-TRACEABILITY
+
+    The plugin shall allow for splitting the documentation in parts.
+
+.. item:: RQT-DOCUMENTATION_ID Identification of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have a unique identification.
+
+.. item:: RQT-CAPTION Brief description of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have a optional brief description.
+
+.. item:: RQT-CONTENT Conten of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have optional content. The content shall be parseable RST, and passed
+    through the configured sphinx parser/renderer.
+
+.. item:: RQT-ATTRIBUTES Documentation parts can have attributes
+    :depends_on: RQT-ITEMIZE
+
+    Attributes shall be able to be added to the documentation parts.
+    Attributes have a key and an optional value.
+    The set of attributes and the validness of the attribute values shall be configurable.
+
+.. item:: RQT-RELATIONS Documentation parts can be linked to each other
+    :depends_on: RQT-ITEMIZE
+
+    Documentation parts shall be able to link to other documentation parts.
```

Comment:
`shall be linked` ( remove `be able to`). Maybe re-reading this couple of times we are missing something like: `This link is optional.` as a next sentence, to show that it is not a required attribute (although might not be applicable for this case)

Reply:
rewording in 792c66b